v2/services/authorize/service_config.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
How is this used?

Reply:
It is loaded as the default service config using `grpc.WithDefaultServiceConfig`. It specifies a retry policy for all gRPC methods.